spec/indexers/hyrax/generic_type_indexer_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe Hyrax::GenericTypeIndexer do
+  let(:user) { create(:user) }
+  let(:admin_set) { AdminSet.new(title: ['Some admin set']) }
```

Comment:
How about changing these defining lets into `build(:admin_set)`, `build(:work)`, `build(:collection)`, and `build(:file_set)`?  You don't appear to be using `title` so a simple `build` with no args besides factory should work I think.